### ISSUE TITLE
Cannot use Yad2 search. Who the hell blocked "yad2.co.il##.search_banners"

### ISSUE THREAD

**Issue by PenguinIsrael** [easylist/EasyListHebrew#247]
I'm not sure if that filter is to blame on the easyliat hebrew, but using ublock origin I can't use the search.
![2017-04-03 20-35-34](https://cloud.githubusercontent.com/assets/5622266/24759938/baaac956-1aef-11e7-80f3-1c3e2ff52281.png)
![2017-04-06 17-31-13](https://cloud.githubusercontent.com/assets/5622266/24759937/baa55cf0-1aef-11e7-8043-d8ed8a61a4f7.png)

Pics for example with and without ublock origin, the drop lists are missing while using it.
I really like your addon, but, how did that happen?

**Comment by IsraeliAdblocker** [comment]
https://github.com/easylist/EasyListHebrew/commit/7d925c85af38615bb8a8a3bf48743caf51554957

**Comment by PenguinIsrael** [comment]
Not sure what to do, still can't use the drop lists.

**Comment by IsraeliAdblocker** [comment]
Ublock Origin takes time to update after a commit, please try now, update the lists and purge cache, it should work now, I've just checked.

**Comment by shmizan** [comment]
Hi.
I purged cache and updated the lists. 
still get the error above.

**Comment by PenguinIsrael** [comment]
firefox's cache or ublock origin's cache?

**Comment by PenguinIsrael** [comment]
same as shmizan (can we talk in hebrew? יכול להיות נחמד)
I've unchecked the list, updated, rechecked it, update. still can't use the drop lists with ublock origin on.

**Comment by shmizan** [comment]
thanks for trying to help.
unfortunately, the new version still does not work.
I've tested with https://raw.githubusercontent.com/easylist/EasyListHebrew/master/EasyListHebrew.txt
to make sure it's the list that causes the problem - I've replaced my filters with the list and disabled ISR from ublock origin's 3rd party filters page, and the problem persisted.

then I've removed 
**||yad2.co.il/js-new/showAds.js$script,redirect=silent-noeval.js,domain=yad2.co.il**
from the list, and it worked as it should..
